renderer/styles/style.css

```diff
@@ -824,11 +824,13 @@ span.qc-option-txt{
   width: 100%;
   height: 100%;
   font-family: NotoSans;
+  padding: 10px 0 0 0;
 }
 .mcq-popup-image {
-  width: 50%;
+  width: 35%;
+  height: 40vh;
```

Comment:
This is not right always.

Reply:
previously height was not there. So, the div is taking default height of the image. I have added height to the div  so that the image will always fit inside the div